rmutil/test_util.h

```diff
@@ -19,6 +19,8 @@
                 
 #define RMUtil_Assert(expr) if (!(expr)) { fprintf (stderr, "Assertion '%s' Failed\n", __STRING(expr)); return REDISMODULE_ERR; }
 
+#define RMUtil_AssertNullReply(rep) RMUtil_Assert(RedisModule_CreateStringFromCallReply(rep) == NULL)
```

Comment:
Why not checking ReplyType == REDISMODULE_REPLY_NULL

Reply:
in order to keep with the convention of a self-contained assert, just like `RMUtil_AssertReplyEquals` below

Reply:
that said, added for compatibility sake